lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -391,6 +395,41 @@ func (handler *ContainersHandlersImpl) ContainerWaitHandler(params containers.Co
 	}
 }
 
+func (handler *ContainersHandlersImpl) RenameContainerHandler(params containers.ContainerRenameParams) middleware.Responder {
+	defer trace.End(trace.Begin(fmt.Sprintf("Rename container to %s", params.Name)))
+
+	h := exec.GetHandle(params.Handle)
+	if h == nil || h.ExecConfig == nil {
+		return containers.NewContainerRenameNotFound()
+	}
+
+	// get the indicated container for rename
+	container := exec.Containers.Container(h.ExecConfig.ID)
+	if container == nil {
+		return containers.NewContainerRenameNotFound()
+	}
+
+	if container.ExecConfig.Name == params.Name {
+		err := &models.Error{
+			Message: fmt.Sprintf("renaming a container with the same name as its current name"),
```

Comment:
are you sure you want this error without any parameters?  in this case no need to call Sprintf at all, it is simply a string.